kube-merit-updater

```diff
@@ -25,6 +25,11 @@ role=''
 resume=''
 timeout=600
 
+# Static vars
+storage_ns='sys-longhorn'
+pvc_label_key='type'
```

Comment:
I happened upon this label today: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/kube-apiserver.manifest#L12

But can also use type, no preference.

Reply:
hmm tier sounds a bit better, as type is usually used more strictly. I will update